LibreHardwareMonitor/UI/MainForm.cs

```diff
@@ -267,7 +268,7 @@ public MainForm()
             celsiusMenuItem.Checked = _unitManager.TemperatureUnit == TemperatureUnit.Celsius;
             fahrenheitMenuItem.Checked = !celsiusMenuItem.Checked;
 
-            Server = new HttpServer(_root, _settings.GetValue("listenerPort", 8085));
+            Server = new HttpServer(_root, _settings.GetValue("listenerPort", 8085), _settings.GetValue("authenticationEnabled", false), _settings.GetValue("httpUsername", "librehm"), _settings.GetValue("httpPassword", "root"));
```

Comment:
These defaults seem unnecessary, I think it makes sense to set the username and password entirely to null/empty unless the user manually specifies anything.